consensus/istanbul/uptime/config.go

```diff
@@ -24,32 +24,38 @@ const (
 // ComputeLookbackWindow computes the lookbackWindow based on different required parameters.
 // getLookbackWindow represents the way to obtain lookbackWindow from the smart contract
 func ComputeLookbackWindow(epochSize uint64, defaultLookbackWindow uint64, isDonut bool, getLookbackWindow func() (uint64, error)) uint64 {
-	if !isDonut {
-		return defaultLookbackWindow
-	}
+	var returnValue uint64
```

Comment:
Why `returnValue` rather than `value`?  Also, we could put in the signature `(value uint64)`